db/models.py

```diff
@@ -1 +1,62 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    description = models.TextField(
+        blank=True
+    )
+
+
+class Skill(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    bonus = models.CharField(
+        max_length=255
+    )
```

Comment:
The `bonus` field is a CharField. If it represents a numerical value, consider changing it to a more appropriate field type, such as IntegerField or FloatField.